### PR TITLE
[Merged by Bors] - chore: backports for leanprover/lean4#4814 (part 31)

### PR DESCRIPTION
**Pull request by kim-em**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/CategoryTheory/Triangulated/Yoneda.lean [hunk 1]
```diff
@@ -48,7 +48,7 @@ lemma preadditiveYoneda_map_distinguished
 noncomputable instance (A : Cᵒᵖ) : (preadditiveCoyoneda.obj A).ShiftSequence ℤ :=
   Functor.ShiftSequence.tautological _ _
 
-lemma preadditiveCoyoneda_homologySequenceδ_apply
+lemma preadditiveCoyoneda_homologySequenceδ_apply {C} [Category C] [Preadditive C] [HasShift C ℤ]
```
Comment (sgouezel) [inline Mathlib/CategoryTheory/Triangulated/Yoneda.lean:51]:
```suggestion
lemma preadditiveCoyoneda_homologySequenceδ_apply {C : Type*} [Category C] [Preadditive C] [HasShift C ℤ]
```

**Review by sgouezel — COMMENTED** [review]
bors d+

**Comment by semorrison** [comment]
bors merge

**Comment by semorrison** [comment]
bors merge

**Pull request closed without merging**